lib/widgets/auth/auth_page.dart

```diff
@@ -159,69 +161,78 @@ class PlatformSignInButton extends StatelessWidget {
   }
 }
 
-class OtherOptionsButton extends StatelessWidget {
+class OtherDefaultProviderButton extends StatelessWidget {
```

Comment:
Just replaced a normal button (with icon and text) that navigated to a page with a single sign in button to a FAB that does the sign in directly

Reply:
nice!!!